gordo_components/model/factories/lstm_autoencoder.py

```diff
@@ -262,6 +261,7 @@ class (e.x. Adam(lr=0.01,beta_1=0.9, beta_2=0.999)).  If no arguments are
 
     return lstm_symmetric(
         n_features,
+        n_features_out,
```

Comment:
Here there is definitely kind of an issue. If you run hourglass with 40 features in, 20 out and compression_factor=0.5, how small would you expect the smallest layer to be? Today it will be 20, so the 20 out nodes can be pushed right through. To me it definitely seems reasonable to have the compression-factor multiply the *smallest* of in and out-nodes. But some care must (maybe) be taken in ensuring that it has the right nr of in-neurons as well, and that the slope from 40 to the smallest layer of 10 is reasonable, and that the slope from 10 to 20 is reasonable. It can definitely no longer use `lstm_symmetric` directly:-p
But is this maybe an imidately new PR/issue?

Reply:
https://github.com/equinor/gordo-components/issues/399